### PR TITLE
Feature/aoaf 478 update check your answers page

### PR DESCRIPTION
**Pull request by wdhda-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/routers/handlers/check_details/index.ts [hunk 1]
```diff
@@ -33,12 +40,39 @@ export class CheckDetailsHandler extends GenericHandler<CheckDetailsViewData> {
...
+
+        // if isBusinessRegistered is true; company name need to be set
+        if (details.companyName === undefined && details.isBusinessRegistered) {
+            throw new Error("Presenter account company name has not been set for a registered business.");
+        }
+
+        // if isBusinessRegistered is false; business name need to be set
+        if (details.businessName === undefined && !details.isBusinessRegistered) {
+            throw new Error("Presenter account business name has not been set for a registered business.");
+        }
```
Comment (amartin7663) [inline src/routers/handlers/check_details/index.ts:60]: Could we encapsulate this in it's own function, this is adding quite a lot of lines to the function

File: src/routers/handlers/confirm_company/index.ts [hunk 2]
```diff
@@ -35,6 +35,8 @@ export class ConfirmCompanyHandler extends GenericHandler<ConfirmCompanyViewData
 
         const { companyStatus: status, dateOfCreation: incorporatedOn, companyName } = await this.getCompanyProfile(req, companyNumber);
 
+        this.setCompanyName(req, companyName);
```
Comment (amartin7663) [inline src/routers/handlers/confirm_company/index.ts:38]: Should we be setting in the GET, as technically the user hasn't confirmed this is the name they want

**Review by amartin7663 — APPROVED** [review]

**Pull request merged**